config/global.ini.php

```diff
@@ -385,6 +385,14 @@
 ; This feature will not work with the MYSQLI extension.
 archiving_query_max_execution_time = 7200
 
+
+; Allow you to disable archiving segment on selected plugins
+; When this is on, it won't archive any data for selected plugin if segment is not empty
+; This field accept split by comma eg: disable_archiving_segment_for_plugin="Plugin1,Plugin2,Plugin3"
```

Comment:
Maybe better "This field accepts more than one plugin separated by comma eg:..."